ironfish/src/mining/stratum/messages.ts

```diff
@@ -137,6 +140,7 @@ export const MiningStatusSchema: yup.ObjectSchema<MiningStatusMessage> = yup
     addressStatus: yup
       .object({
         publicAddress: yup.string().required(),
+        activeMiners: yup.string().required(),
```

Comment:
could this be an array of strings instead? then it wouldn't be necessary to stringify the list below

Reply:
Yeah, you can do that in YUP like this...
```suggestion
        activeMiners: yup.array(yup.string().required()).required(),
```